lib/core/presentation/widgets/post/post_profile_card_widget.dart

```diff
@@ -58,24 +63,31 @@ class PostProfileCard extends StatelessWidget {
                 Row(
                   mainAxisSize: MainAxisSize.max,
                   children: [
-                    Text(postUsername),
+                    Text(postName,
+                        style: Typo.medium
+                            .copyWith(fontFamily: FontFamily.circularStd)),
```

Comment:
The design for home screen not update the font family yet, please don't use this font. 
Base font already setup in theme

Reply:
btw, please add the comma